bpo-35923: Update the BuiltinImporter to use loader._ORIGIN (#15651)
python/cpython: Misc/NEWS.d/next/Library/2019-09-03-01-41-35.bpo-35923.lYpKbY.rst

> Update the BuiltinImporter in importlib to use loader._ORIGIN instead of a

Reviewer: Instead of having to specify the class is located within importlib, we can simply provide an inline link to its [documentation](https://docs.python.org/3/library/importlib.html#importlib.machinery.BuiltinImporter) using Sphinx markup.

Since ``loader._ORIGIN`` is internal, we can use inline code blocks for formatting instead. 

```suggestion
Update :class:`importlib.machinery.BuiltinImporter` to use ``loader._ORIGIN`` instead of a
```
